install/upgrade/manual/update_fail2ban-dovecot-whitelist.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# info: enable fail2ban dovecot whitelisting
+#
+# The function:
+#   edits jail.local
+#   adds action.d/ignoreip.conf
+#   adds filter.d/dovecot-whitelist.conf
+
+#----------------------------------------------------------#
+#                    Variable&Function                     #
+#----------------------------------------------------------#
+
+#----------------------------------------------------------#
+#                    Verifications                         #
+#----------------------------------------------------------#
+
+#----------------------------------------------------------#
+#                       Action                             #
+#----------------------------------------------------------#
+
+#Adds the config lines at the end of jail.local
+file1='/etc/fail2ban/jail.local'
```

Comment:
```suggestion
file='/etc/fail2ban/jail.local'
```
or below use `$file1` instead of `$file`

Reply:
Oh my God! You are right!